### PR TITLE
Add Other Approaches paragraph to literature review

### PR DESCRIPTION
**Pull request by dmurvihill**

### CODE REVIEW

File: doc/src/bibliography.bib [hunk 1]
```diff
@@ -38,11 +38,11 @@ @inproceedings{belkhouche04
 	year={2004},
 	organization={ACM},
 	annote={
-    \setlength{\parskip}{1.5ex}
-	  Unsuitable for use in project---focuses on very high-level comparison,
-	  analyzing design (code structure, data structures) of two programs for
-	  similarity (overall program design is analyzed and compared---going to be
-	  identical in low-level CS!). Mention at most.}
+          \setlength{\parskip}{1.5ex}
```
Comment (mheon) [inline doc/src/bibliography.bib]: Don't indent the setlength

File: doc/src/bibliography.bib [hunk 2]
```diff
@@ -584,9 +582,9 @@ @article{lancaster05
   publisher={The Higher Education Academy Innovation Way, York Science Park, Heslington, York YO10 5BR},
   annote={
     \setlength{\parskip}{1.5ex}
-    TODO. Entirely devoted to producing a taxonomy of plagiarism detection solutions ---
-    existing software, types of detection engine, etc. Going to be VERY useful
-    writing literature survey.}
+    Lancaster provides a detailed taxonomy of plagiarism detection tools, and
+    describes all (then) commonly-known plagiarism detection tools in terms of
```
Comment (mheon) [inline doc/src/bibliography.bib]: Don't like the repetition of "plagiarism detection tools" - maybe change the second to "commonly-known examples"?

File: doc/src/litreview.ltx [hunk 3]
```diff
@@ -422,14 +422,29 @@ pursue this line of inquiry.
...
+effectiveness of the Burrows approach requires using an intelligent similarity
+scoring function, a number of such functions have been developed, including the
+Okapi BM25 function and a family of functions developed by Ciesielski, Nelson,
+and Tahaghogi using genetic programming \cite{ciesielski08}.
+
+Belkhouche, Nix, and Hassell have contributed an elaborate academic dishonesty
+detection approach that compares C programs by converting them to structures
+representing control flow, data tables, and other high-level concepts. Their
+implementation does not compare implementations written in other programming
+languages \cite{belkhouche04}.
```
Comment (mheon) [inline doc/src/litreview.ltx]: Might be better to say it's very language specific?

File: doc/src/bibliography.bib (hunk 1 above)
Comment (mheon) [inline doc/src/bibliography.bib]: Or alternatively also indent the next line. Which seems to be more consistent?

Comment (dmurvihill) [inline doc/src/bibliography.bib]: Replaced tab indentation with space indentation and made the indentation work better.

**Comment by dmurvihill** [comment]
fixed your comments on this item.

**Comment by mheon** [comment]
Looks good, merging

**Pull request merged**